.github/workflows/minikube.yaml

```diff
@@ -18,9 +18,14 @@ jobs:
         run: kubectl get nodes
       - name: Download installer dependencies
         run: |
+          curl -sSL -o /tmp/vault.zip https://releases.hashicorp.com/vault/1.6.2/vault_1.6.2_linux_amd64.zip
+          unzip /tmp/vault.zip
+          sudo mv vault /usr/local/bin/vault
```

Comment:
You could do this with `install -m0755 vault /usr/local/bin` but I guess that wouldn't do the remove, so, eh, it's fine.

Reply:
Yeah, kind of a bummer, maybe later on we can wrap that into a setup script that would make the virtualenv later on